### DISCUSSION TITLE
Spiral MATE edition - canberra gtk modules required

### DISCUSSION THREAD

**Discussion by bin-linux** [SpiralLinux/SpiralLinux-project#104] [General]
Out of the box MATE is throwing errors 'failed to load canberra-gtk-module' throughout .xession-errors

To remove these you need to install libcanberra-gtk3-module libcanberra-gtk-module

**Comment by geckolinux** [comment]
OK, great, thanks for the tip. Looks like a small but helpful edition for the next release.

**Comment by geckolinux** [comment]
Fixed in SpiralLinux 11.220925 release.
